src/utils/generateFlowDiagramData.js

```diff
@@ -97,14 +98,25 @@ const generateFlowDiagramData = (data) => {
     { flowData: {}, mostRecentYear: -Infinity, mostRecentMonth: -Infinity }
   );
 
-  Object.values(flowData).forEach((item) => {
-    const hint = item.isNotAvailable
+  Object.values(flowData).forEach((metric) => {
+    const hint = metric.isNotAvailable
       ? null
-      : generateHint(mostRecentYear, mostRecentMonth, item.item);
+      : generateHint(mostRecentYear, mostRecentMonth, metric.item);
 
     if (hint) {
       // eslint-disable-next-line no-param-reassign
-      item.hint = hint;
+      metric.hint = hint;
+    }
+
+    const { item } = metric;
+
+    const isTooStale = metric.isNotAvailable
+      ? null
+      : item.month < mostRecentMonth && item.year <= mostRecentYear - 1;
```

Comment:
It would be nice to create a little utility function, `metricIsTooStale`,  that takes in a metric/item and the most recent month and year and returns a boolean. Then we would write unit tests for that function.

Reply:
Done